base/applications/cleanmgr/dialog.c

```diff
@@ -0,0 +1,360 @@
+/*
+ * PROJECT:         ReactOS Disk Cleanup
+ * LICENSE:         GPL - See COPYING in the top level directory
+ * PURPOSE:         Dialog Procs
+ * COPYRIGHT:       Copyright 2020 Arnav Bhatt (arnavbhatt288 at gmail dot com)
+ */
+ 
+#include "resource.h"
+#include "precomp.h"
+
+#define CX_BITMAP 18
+#define CY_BITMAP 18
+
+DIRSIZE sz;
+DLG_VAR dv;
+WCHAR_VAR wcv;
+BOOL_VAR bv;
+
+BOOL CALLBACK StartDlgProc(HWND hwnd, UINT Message, WPARAM wParam, LPARAM lParam)
+{
+	WCHAR LogicalDrives[MAX_PATH] = { 0 };
+	int ItemIndex = 0;
+	static HBITMAP hbmDrive;
+	static HBITMAP hbmMask;
+	DWORD dwIndex = 0;
+
+	bv.sysDrive = FALSE;
+	
+	DWORD drives = GetLogicalDriveStringsW(MAX_PATH, LogicalDrives);
+	if (drives == 0)
+	{
+		MessageBoxW(NULL, L"GetLogicalDriveStringsW() failed!", L"Error", MB_OK | MB_ICONERROR);
```

Comment:
This code is always executed for each procedure call. Move to the specific place.